builder/builder-next/builder.go

```diff
@@ -346,14 +357,19 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 		}
 	}
 
-	if exporterName == "moby" {
-		if len(opt.Options.Tags) > 0 {
-			exporterAttrs["name"] = strings.Join(opt.Options.Tags, ",")
+	if (exporterName == client.ExporterImage || exporterName == mobyexporter.Moby) && len(opt.Options.Tags) > 0 {
+		reposAndTags, err := mobyexporter.SanitizeRepoAndTags(opt.Options.Tags)
+		if err != nil {
+			return nil, err
+		}
+		names := strings.Join(reposAndTags, ",")
+		exporterAttrs["name"] = names
+		if b.useSnapshotter {
+			exporterAttrs["image.name"] = names
```

Comment:
Does this need both of these attributes?  (The latter appears unused, unless it's passed along to another library :see_no_evil:)

Reply:
Good catch, only `"name"` is needed here